### PR TITLE
feat: a WP plugin to load scripts needed for read along web app 

### PR DESCRIPTION
**Pull request by deltork**
This is a plugin that will allow WordPress sites to load the necessary script for the Read Along web app component

### CODE REVIEW

File: packages/web-component/src/components.d.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@
  */
 import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 import { InterfaceLanguage, ReadAlongMode } from "./index.d";
-export { InterfaceLanguage, ReadAlongMode } from "./index.d";
```
Comment (joanise) [inline packages/web-component/src/components.d.ts:9]: AP's recent PR had this line automatically inserted, now this PR automatically removes it. I wonder if we might be working with different versions of node or something. Did you recently redo `npm install` in the project?

File: packages/studio-web/src/app/demo/demo.component.ts [hunk 2]
```diff
@@ -323,8 +323,45 @@ Please host all assets on your server, include the font and package imports defi
...
+
+        ----- END OF SNIPPET----
+
+        
+        `,
+      ]);
+
       // - add plain text readme
-      innerFolder?.file("README.txt", this.readmeFile);
+      innerFolder?.file("README.txt", WP_deployment_readme);
```
Comment (joanise) [inline packages/studio-web/src/app/demo/demo.component.ts:364]: I'm having a hard time deciding if I prefer one README.txt having both the regular web deployment guide and the WordPress deployment guide (as you did here), or if I want two separate files. 

But I think this is good as is.

**Review by joanise — APPROVED** [review]
This is good. Thanks!

File: packages/studio-web/src/app/demo/demo.component.ts (hunk 2 above)
Comment (deltork) [reply packages/studio-web/src/app/demo/demo.component.ts:364]: Same with me, so I set it so that we can separate them easily if we want.

**Pull request merged**